tests/test_find.py

```diff
@@ -35,26 +35,43 @@
 from tuna.sql import DbCursor
 from tuna.utils.logger import setup_logger
 from tuna.find_db import ConvolutionFindDB
-
-
-def add_job():
-  find_configs = "SELECT count(*), tag FROM conv_config_tags WHERE tag='recurrent_pytest' GROUP BY tag"
-
-  del_q = "DELETE FROM conv_job WHERE reason = 'tuna_pytest'"
-  ins_q = "INSERT INTO conv_job (config, state, solver, valid, reason, session) \
-        SELECT conv_config_tags.config, 'new', NULL, 1, 'tuna_pytest', 1 \
-        FROM conv_config_tags WHERE conv_config_tags.tag LIKE 'recurrent_pytest'"
-
-  with DbCursor() as cur:
-    cur.execute(find_configs)
-    res = cur.fetchall()
-    if len(res) == 0:
-      add_cfg = "{0}/../tuna/import_config.py -f {0}/../utils/recurrent_cfgs/alexnet_4jobs.txt -t recurrent_pytest -V 1.0.0".format(
-          this_path)
-      os.system(add_cfg)
-
-    cur.execute(del_q)
-    cur.execute(ins_q)
+from load_job import test_tag_name as tag_name_test, add_jobs
+from utils import CfgImportArgs, LdJobArgs
+from tuna.db_tables import connect_db
+from tuna.tables import DBTables
+from tuna.import_configs import import_cfgs
+
+
+def add_job(arch, num_cu):
+
+  #import configs
+  session = 1
```

Comment:
ditto, creation / existence of the session should be assured by the pytest.

Reply:
seems like this part of the code is not needed in this utest. Ill remove